extensions/rt_ignore_dh/rt_ignore_dh.c

```diff
@@ -177,27 +182,37 @@ static int stow_destination_host(struct msg **msg) {
 }
 
 /* The callback for putting Destination-Host into Proxy-Info and restoring it on the way back */
-static int rt_ignore_destination_host(void * cbdata, struct msg **msg) {
+static void rt_ignore_destination_host(enum fd_hook_type type,
```

Comment:
I'm not sure we should change the existing module.  Shouldn't we create a new one?